Neos.Neos/Classes/Service/XliffService.php

```diff
@@ -127,6 +127,19 @@ public function getCachedJson(Locale $locale)
         return $json;
     }
 
+    /**
+     * @return integer The current cache version identifier
+     */
+    public function getCacheVersion()
```

Comment:
```suggestion
    public function getCacheVersion(): string
```
That should return a string as a string casted version is written to the cache below